File: 151002/Haiduk/RV/RV/Lab2/DTO/ResponseDTO/NewsResponseDto.cs
using Lab2.Models;

namespace Lab2.DTO.ResponseDTO
{
    public class NewsResponseDto
    {
        public long Id { get; set; }
        public long CreatorId { get; set; }
        public Creator Creator { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }
    }
}

File: 151001/Maliauka/src/main/java/by/bsuir/dc/features/news/dto/NewsResponseDto.java
package by.bsuir.dc.features.news.dto;

import by.bsuir.dc.features.news.News;

import java.io.Serializable;
import java.time.Instant;
import java.util.List;

/**
 * DTO for {@link News}
 */
public record NewsResponseDto(Long id, String title, String content, Instant created, Instant modified, List<Long> postIds,
                              Long editorId) implements Serializable {
}